src/utils/createStore.js

```diff
@@ -17,6 +17,13 @@ import * as appReducer from 'state';
 
 import { initialState } from 'state/initialState';
 
+const createHistoryWithBasename = (historyOptions) => {
+  return useBasename(createHistory)({
+    basename: location.hostname === 'localhost' ? '/' : '/i-have-to-return-some-videotapes',
```

Comment:
That doesn't handle the case of landing on `/i-have-to.../editor/` though. Also a better ternary would be like a webpack-provided one at build-time